### PR TITLE
Update readme and add new example

### PR DESCRIPTION
**Pull request by ellisjoe**

### CODE REVIEW

**Comment by ellisjoe** [comment]
@uschi2000 @markelliot planning on leaving hadoop-crypto as the repo name and for artifact publishing out of simplicity. don't think it gets us anything to rename at the moment. will likely 2.0 after these docs changes go in

File: README.md [hunk 1]
```diff
@@ -1,6 +1,57 @@
 [![CircleCI Build Status](https://circleci.com/gh/palantir/hadoop-crypto/tree/develop.svg?style=shield)](https://circleci.com/gh/palantir/hadoop-crypto)
 [![Download](https://api.bintray.com/packages/palantir/releases/hadoop-crypto/images/download.svg)](https://bintray.com/palantir/releases/hadoop-crypto/_latestVersion)
 
+Seekable Crypto
+===============
+*Seekable Crypto* is a Java library that provides the ability to seek within
+`SeekableInput`s while decrypting the underlying contents along with some
+utilities for storing and generating the keys used to encrypt/decrypt the data
+streams. An implementation of the Hadoop FileSystem is also included that uses
+the Seekable Crypto library to provide a efficient and transparent client-side
```
Comment (uschi2000) [inline README.md]: -a

File: README.md [hunk 2]
```diff
@@ -1,6 +1,57 @@
...
+encryption.
```
Comment (uschi2000) [inline README.md]: ... encryption for Hadoop filesystems.

**Review by uschi2000 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ellisjoe) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (ellisjoe) [reply README.md]: done

**Comment by uschi2000** [comment]
+1

**Pull request merged**